helm/dagster/templates/deployment-celery-queues.yaml

```diff
@@ -85,6 +85,9 @@ spec:
             - name: dagster-celery
               mountPath: "{{ $.Values.global.dagsterHome }}/celery-config.yaml"
               subPath: celery.yaml
+            {{- range $volumeMount := $celeryK8sRunLauncherConfig.volumeMounts }}
```

Comment:
Generally the user doesn't actually care if we stick a mount on the celery worker right? But I think it can't hurt

Reply:
fair point but then why are the envSecrets and envConfigmaps here?

Reply:
yeah, not clear